### PR TITLE
Use SDK generated docs

### PR DESCRIPTION
**Pull request by will-v-pi**
Just a draft for now, as I haven't fixed the theming, but I was thinking it might be good to switch to using the documentation generated by the SDK as I think that will be easier to navigate, and can be generated for each new sdk version (maybe even showing a different version depending on the SDK version in use). Do you think this looks alright?

One issue is that there are a lot of files needed to make this documentation, so would that cause an issue with distributing the extension?

### CODE REVIEW

**Comment by paulober** [comment]
These docs look so much better.

File: src/commands/openSdkDocumentation.mts [hunk 1]
```diff
@@ -88,17 +98,30 @@ export default class OpenSdkDocumentationCommand extends CommandWithArgs {
...
+    panel.onDidDispose(
+      event => {
+        this._panel = undefined;
+      }, this
+    );
   }
 
   private _getHtmlForWebview(
     url: string,
     docstyle: string,
```
Comment (paulober) [inline src/commands/openSdkDocumentation.mts:117]: docstyle can be removed as you deleted this stylesheet.

**Comment by paulober** [comment]
> One issue is that there are a lot of files needed to make this documentation, so would that cause an issue with distributing the extension?

No, that's not a problem but it would increase the extension bundle by 11MB.

**Comment by paulober** [comment]
> and can be generated for each new sdk version (maybe even showing a different version depending on the SDK version in use).

I just thought that we could run the docs generation on sdk installation so we don't have to distribute them. But this would require another binary dependency.

So maybe just include the most recent versions.

**Comment by paulober** [comment]
Should we now merge it as light mode only, for the beta?

**Comment by will-v-pi** [comment]
Yeah, I'm happy to merge this now

**Review by paulober — APPROVED** [review]

File: src/commands/openSdkDocumentation.mts (hunk 1 above)
Comment (paulober) [reply src/commands/openSdkDocumentation.mts]: Fixed with 55d049978bdd50336b77d7dbbdb50fde4473dccb

**Pull request merged**